src/mock/v1.2/channelService-1_2.ts

```diff
@@ -79,14 +79,14 @@ export class Fdc3CommandExecutor1_2 {
 
   //app channel broadcast service
   appChannelBroadcastService = {
-    broadcast: (contextType, historyItems, channel, testId) => {
+    broadcast: (contextType: string, historyItems: number, channel: Channel, testId: string) => {
       if (channel !== undefined) {
```

Comment:
There should be an else clause here throwing an error if that channel is undefined. Otherwise this is a noop